### DISCUSSION TITLE
Forward sensor not adding to grid in distributed deployment

### DISCUSSION THREAD

**Discussion by sam-acker** [Security-Onion-Solutions/securityonion#5037] [Unsupported Versions] [answered]
Have successfully deployed a manager node and a search node (both of which show up in the grid in the manager node's web console), however I am having trouble deploying a forward node. The installation of the forward node seems to be successful as there were no reported errors and the logs appear to show none.
[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/6919887/sosetup.log)
<img width="572" alt="Screen Shot 2021-08-02 at 4 11 18 PM" src="https://user-images.githubusercontent.com/37353790/127930463-78bc5184-2944-412f-8505-e059e3d7590f.png">

`so-status` shows everything ok on both the manager and this new forward node. Also `salt-key -l all` lists the key of the forward node as being accepted. Both the forward node and the manager node can ping each other, firewall rules in the router have been configured to allow all traffic, and I used `sudo so-firewall includehost minion` and `sudo so-firewall includehost sensor` to explicitly allow the sensor. 

At a loss here on how to troubleshoot further. Please let me know if there's any other info or logs I can provide, would really appreciate some help on this. Thanks!

**Comment by TOoSmOotH** [comment]
Is it in grid? Is there data in hunt?

**Reply by sam-acker** [reply]
No, it is not in the grid. We're working in a proxmox virtual environment and we were able to get everything to work by making the management NIC of the forward node on the same subnet as the manager node (10.0.2.0/24) and keeping the tap NIC on the network we're trying to monitor (10.0.1.0/24). However, we're not able to see the forward node in the grid when its management NIC has an ip in the 10.0.1.0/24 range or in the 192.168.0.0/16 range which is the WAN for both 10.0.2.0/24 and 10.0.1.0/24. Whenever the forward node is in either the 10.0.1.0/24 or 192.168.0.0/16 ranges it successfully installs and the salt key gets added to the manager, we can even see it in Grafana, but it doesn't show up in the grid and won't send data to kibana/hunt.

**Comment by sam-acker** [comment, marked as answer]
Solved this using a site to site VPN between the network with the manager node and the network with the forward node.
